default.nix

```diff
@@ -1,5 +1,23 @@
 { # The files would be going to ~/.config/doom (~/.doom.d)
```

Comment:
I think a check in the flake.nix could be nice; `init-example-el-splitdir` or something.

Reply:
Yeah, I coucur. Would be nice to add a test for this case.

Reply:
Good idea! Check the test I just added.